exchanges/coinbasepro/coinbasepro.go

```diff
@@ -162,19 +162,28 @@ func (c *CoinbasePro) GetTrades(currencyPair string) ([]Trade, error) {
 
 // GetHistoricRates returns historic rates for a product. Rates are returned in
 // grouped buckets based on requested granularity.
-func (c *CoinbasePro) GetHistoricRates(currencyPair string, start, end, granularity int64) ([]History, error) {
+func (c *CoinbasePro) GetHistoricRates(currencyPair string, start, end string, granularity int64) ([]History, error) {
```

Comment:
You can drop the string after `currencyPair` now

Reply:
done